codestyle-guide.md

```diff
@@ -0,0 +1,90 @@
+# Grafana Plugins Code Style Draft
+
+## Repo name
+It's a good practice to use plugin repo name same as plugin id.  
+Plugin id has following structure:
+```
+(username|company)-(plugin name)-(datasource|app|panel)
```

Comment:
This needs to be `(grafana.net username or org slug)-(plugin name)-(datasource|app|panel)`